en/core-libraries/events.rst

```diff
@@ -466,6 +466,26 @@ params you used for attaching it::
     // Detaching all callbacks implemented by a listener
     $this->getEventManager()->detach($listener);
 
+Core events
+===========
+
+There are a number of core events within the framework which your application can listen to.
+
+* beforeMarhsal
+* beforeFind
+* buildValidator
+* buildRules
```

Comment:
Out of topic but maybe we should add a table in the README to inform potential contributors what branch is dedicated to which version of the docs ? It seems to be a commonly made mistake. I agree it will not prevent people from making it again but at least this will be stated somewhere.